.github/workflows/test.yml

```diff
@@ -32,10 +34,60 @@ jobs:
       uses: actions-rs/cargo@v1
       with:
         command: clippy
+
   build-image:
     name: Test the build of Docker image
     runs-on: ubuntu-latest
     steps:
     - uses: actions/checkout@v1
     - run: docker build . --file Dockerfile -t meilisearch
       name: Docker build
+
+  ## A push occurred on a release branch, a prerelease is created and assets are generated
+  prerelease:
+    name: create prerelease
+    needs: [check, build-image]
+    if: ${{ contains(github.ref, 'release-') && github.event_name == 'push' }}
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+      - name: Get version number
+        id: version-number
+        run: echo "##[set-output name=number;]$(echo ${{ github.ref }} | sed 's/.*\(v.*\)/\1/')"
+      - name: Get commit count
+        id: commit-count
+        run: echo "##[set-output name=count;]$(git rev-list remotes/origin/master..remotes/origin/release-${{ steps.version-number.outputs.number }} --count)"
+      - name: Create Release
+        id: create_release
+        uses: actions/create-release@v1
+        env:
+          GITHUB_TOKEN: ${{ secrets.PUBLISH_TOKEN }} # PAT
```

Comment:
what does PAT mean?

Reply:
Public Authentication Token, it's a token that allow application to do things on your behalf (@Kerollmops 's behalf in this case). This action cannot be done with the github token, otherwise, other workflows are not triggered on new releases

Reply:
Personal authentication token. That is the token of one of us if im not wrong 🙂

Reply:
ok thanks, so either we remove the comment because it does not bring clarity or we write the whole words ^^

Reply:
np, PAT is how they refer to it in all their doc